submissions/VikTorNik/froggen-oojs/app.js

```diff
@@ -0,0 +1,106 @@
+const GAME_CONF = {
+    OffsetX : 101,
+    OffsetY : 83,
+    overlap : 0.8,
+    collumn : 5,
+    countWinner : 0,
+    countСheckmate : 0,
+};
+
+const PLAYER_CONF = {
+    startX: 202,
+    startY: 405,
+};
+
+const ENEMY_CONF = {
+    startY: GAME_CONF.OffsetY / 4,
+    speed: 50,
+    maximumCount: 5,    
+};
+
+let allEnemies = [];
+
+let Enemy = function (x, y, speed) {
+    this.sprite = 'images/enemy-bug.png';
+    this.x = x;
+    this.y = y;
+    this.speed = ENEMY_CONF.speed * (1 + Math.random() * 4);
+};
+
+Enemy.prototype.update = function (dt) {
+    this.x += this.speed * dt;
+    this.x > GAME_CONF.OffsetX * GAME_CONF.collumn ? this.x = -GAME_CONF.OffsetX : false;
+    this.collision();
+    this.winner();
+};
+
+Enemy.prototype.render = function () {
+    ctx.drawImage(Resources.get(this.sprite), this.x, this.y);
+};
+
+function createEnems() {
+    allEnemies = [...Array(Math.floor((3 + Math.random() * (ENEMY_CONF.maximumCount - 2))))].map((_, i) =>
+        i = new Enemy(-GAME_CONF.OffsetX, GAME_CONF.OffsetY * Math.floor(Math.random() * 3 + 1) - ENEMY_CONF.startY, ENEMY_CONF.speed));
+};
+
+createEnems();
```

Comment:
Do not intermix class definitions with other code. 
Definitions, then executions.
Tidy your code up.